app/uk/gov/hmrc/cataloguefrontend/metrics/model/ServiceMetricsEntry.scala

```diff
@@ -16,9 +16,14 @@
 
 package uk.gov.hmrc.cataloguefrontend.metrics.model
 
-final case class MetricsEntry (dependency: DependencyName, happyCount: Int, unHappyCount: Int){
+case class ServiceMetricsEntry(
+                           name: String,
+                           count: Int,
+                           happyCount: Int,
+                           unHappyCount: Int
+                         ){
   def asPercentage: List[Int] = {
```

Comment:
It's not intuitive what the returned List is. Would it be better to add derived fields`def happyPercent` `def unhappyPercent`?
Do we need count as well as happyCount/unhappyCount in the API - it can all be derived and adds room for being out-of-sync?

Reply:
> It's not intuitive what the returned List is. Would it be better to add derived fieldsdef happyPercent def unhappyPercent?

agreed. I initially had them as tuples but then I struggled with accessing them in the html template. can have them as separate methods and construct the list of pie values (happy%, unhappy%) explicitly. 

> Do we need count as well as happyCount/unhappyCount in the API? platform metrics does expose `count` which helps sanity check the results like you say. wasn't sure what the benefit of parsing them in catalogue-frontend would be tho. happy to add them in if you feel they'd be useful.